views/templates.pug

```diff
@@ -168,6 +171,31 @@
         button.btn.btn-light.btn-sm.fas.fa-info.full-compiler-name(data-trigger="click" style="cursor: pointer;" role="button" title="Full compiler version")
         span.compile-time(title="Compilation time (Result size)")
 
+  #executor
+    .top-bar.btn-toolbar.bg-light(role="toolbar")
+      .btn-group.btn-group-sm(role="group" aria-label="Compiler picker")
+        select.compiler-picker(placeholder="Select a compiler...")
+        .input-group.mb-auto
+          .input-group-prepend(title="Show all compilation options")
+            button.input-group-text.prepend-options(data-trigger="click" style="cursor: pointer;" role="button" title="All compilation options")
+              span.btn.btn-light.btn-sm.status-icon
+          input.compilation-options.form-control(type="text" placeholder="Compiler options..." size="256" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false")
+          .input-group-append.populararguments(title="Popular arguments")
+            button.btn.btn-outline-secondary.dropdown-toggle.dropdown-toggle-split(type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false")
+              span.sr-only Popular arguments
+            div.dropdown-menu.dropdown-menu-right
+          input.execution-arguments.form-control(type="text" placeholder="Execution arguments..." size="256" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false")
```

Comment:
we should probably add this to the regular compiler pane as well

Reply:
I think we do, but maybe not in this PR? Just the UI changes there scare me already :P

Reply:
oh no not at all, just a reminder